config/environments/development.rb

```diff
@@ -9,13 +9,31 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
-  # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  # Show full error reports.
+  config.consider_all_requests_local = true
+
+  # Enable/disable caching. By default caching is disabled.
```

Comment:
Caching is in het geval van CATja vrijwel altijd wenslijk tijdens development, anders moeten er namelijk bij elk nieuwe antwoord n+1 verzoeken naar OpenCPU gaan.